test/Debugger/JsDiagGetFunctionPosition.js.dbg.baseline

```diff
@@ -31,8 +31,8 @@
   },
   {
     "functionPosition": {
-      "scriptId": 5,
-      "parentScriptId": 4,
+      "scriptId": 6,
+      "parentScriptId": 5,

```

Comment:
i think this should be fine.

---
In reply to: [115389856](https://github.com/Microsoft/ChakraCore/pull/2898#discussion_r115389856) [](ancestors = 115389856)